clients/cli/src/main/java/org/apache/gravitino/cli/GravitinoCommandLine.java

```diff
@@ -476,6 +476,16 @@ protected void handleUserCommand() {
 
     Command.setAuthenticationMode(auth, userName);
 
+    if (CommandActions.LIST.equals(command)) {
+      newListUsers(url, ignore, metalake).handle();
+      return;
+    }
+
+    if (Objects.isNull(user)) {
+      System.err.println("User is undefined. Please use --user to specify a user.");
+      return;
+    }
```

Comment:
I would do this in a different way:
```java
  // A user is mandatory for all actions except for `list`.
  if (Object.isNull(user) && !CommandActions.LIST.equals(command) {
    System.err.println("....");
    return;
  }
```

The handler for `list` can be left as is.